pkg/sql/sqlbase/type_desc.go

```diff
@@ -268,7 +269,7 @@ func (desc *MutableTypeDescriptor) AddEnumValue(node *tree.AlterTypeAddValue) er
 	}
 	if found {
 		if node.IfNotExists {
-			return nil
+			return pgnotice.Newf("enum label %q already exists, skipping", node.NewVal)
```

Comment:
We don't want to return an error here. Instead, we want to use `p.SendClientNotice` and return nil. Also add a test in `pkg/sql/logictest/testdata/logic_test/notice`

Reply:
ohh... thanks for your review.  i have noticed that sqlbase package is getting removed here #52519. Also planner struct is not accessible in sqlbase package. So using p.SendClientNotice  directly doesn't seems feasible to me.
I have just started with this project and i am reading architecture docs in details. So your guidelines will be really helpful for me as a beginner to opensource world as i really want to learn and contribute to this project.

Reply:
> i have noticed that sqlbase package is getting removed here #52519.

This isn't a worry -- all of the code will still be here, the package is just getting split up into smaller packages.

> Also planner struct is not accessible in sqlbase package. So using p.SendClientNotice directly doesn't seems feasible to me.

There are a couple options here:
1. return an error here, and then catch it and convert it into a notice in the statement execution path
2. pass a closure into this function that does the work of sending a notice
3. In the statement (alter_type.go) just check if the enum value exists or not. Then this function can just error if it finds the value already present in the enum.

Overall, I think that the third is the cleanest